controllers/controllers/services/cfservicebinding_controller.go

```diff
@@ -36,21 +36,29 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 -generate
+//counterfeiter:generate -o fake -fake-name VCAPServicesSecretBuilder . VCAPServicesSecretBuilder
+type VCAPServicesSecretBuilder interface {
+	BuildVCAPServicesEnvValue(context.Context, *korifiv1alpha1.CFApp) (string, error)
+}
+
 // CFServiceBindingReconciler reconciles a CFServiceBinding object
 type CFServiceBindingReconciler struct {
 	client.Client
-	Scheme *runtime.Scheme
-	Log    logr.Logger
+	Scheme  *runtime.Scheme
+	Log     logr.Logger
+	Builder VCAPServicesSecretBuilder
```

Comment:
Provided that we have a constructor for this struct, I don't think that we need `Builder` to be public. Same goes to `Scheme` and `Log`

Reply:
Good catch. Fixed